doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
```

Comment:
+1 for hyper(-)parameters in there somehow. (for google)
If it looks too long I'd be +0.5 for removing the word "estimator" from the title.
Another good word that people use is "Tuning"